MIDI Editor/talagan_OneSmallStep/talagan_OneSmallStep Helper lib.lua

```diff
@@ -0,0 +1,126 @@
+-- @noindex
+-- @author Ben 'Talagan' Babut
+-- @license MIT
+-- @description This is part of One Small Step
+
+--[[
+-- Ruby script to generate note FX params
+def putn(note)
+  puts "slider#{3*note     + 10}<0,127,1>Note #{note} Pitch"
+  puts "slider#{3*note + 1 + 10}<0,15,1> Note #{note} Channel"
+  puts "slider#{3*note + 2 + 10}<0,127,1>Note #{note} Velocity"
```

Comment:
REAPER v6.74 added JSFX pre-processing so that JSFX code can generate itself.

https://www.reaper.fm/sdk/js/preproc.php

An example of it used to generate sliders: https://github.com/ReaTeam/JSFX/blob/99510175e9fa087eac446df3b66dedb5e264d4fa/Utility/mschnell_Inter%20Track%20Parameter%20Modulation.jsfx#L26.

Reply:
Missed that one. This is insane 😄 I'm going to give it a try !

Reply:
Done, this opens up a new world ! So cool ! Thanks for pointing me to this worthy feature.